packages/rule-http-cache/README.md

```diff
@@ -65,7 +65,7 @@ validate that the page and resources have a good caching strategy:
 The built-in regular expression for file revving is:
 
 ```regexp
-/\/(\w|-|_)+(\.|-|_)\w+\.\w+$/i
+/\/.+[._-]\w+\.\w+$/i
```

Comment:
I would just check for a number after a`.` or `_` or `-`

```js
/\/.+[._-].*\d.*\.\w+/
```

Reply:
That regex seems to work fine. I've added a few more examples in https://regex101.com/r/6VduJO/5/

@UziTech can you please update your PR with the new link and regex?

Thanks!

Reply:
is this one supposed to match?

```
https://static.xx.fbcdn.net/rsrc.php/v3iJhv4/yG/l/en_US/sqNNamBywvN.js
```

Reply:
Facebook uses another approach to cache busting which is by path instead of file IIRC so that one is supposed to match.

Reply:
in that case just checking that a number exists would be enough

```js

/\d/
```

Reply:
a couple of regex that check for what?

Reply:
This is going to be a tough problem to get everyone to agree on since there are so many ways someone could version a file name.

Off the top of my head:

 - filename.v1.2.3.js (semver)
 - filename-f1leha5h.js (file hash)
 - filename_1234567890.js (last modified time)

Reply:
- /page.php/vp47hv3rs10n/filename.js (path version)

Reply:
>a couple of regex that check for what?

For different patterns.

>This is going to be a tough problem to get everyone to agree on since there are so many ways someone could version a file name.

IIRC the rule can be configured with your own regex, so we don't need to support all of them, just the more commons, and if someone needs an specific regex, they can configure it.

Reply:
what are the most common ones?

Reply:
I don't know hahaha

Reply:
A couple thoughts on how to do this:

* Look into the most common tools for cache busting (`gulp-rev`,  webpack's cache busting with `[hash]` and `[chunkhash]`, etc.)
* Get a list of all the resources from Httparchive that have a significant `max-age` value and then figure out the patterns.

I think that to make it easier to maintain we should have regex per categories, kind of what @UziTech is doing in the comments (semver, file hash, modified time, etc.).

What do you think?